api/cloud/v1.md

```diff
@@ -0,0 +1,116 @@
+    --blue: #007bff;
+    --green: #28a745;
+    --orange: #fd7e14;
+    --yellow: #ffc107;
+    --purple: #6f42c1;
+    --nav-delete-color: var(--red);
+    --nav-post-color: var(--green);
+    --nav-get-color: var(--blue);
+    --nav-put-color: var(--orange);
+  }
+
+  .m-btn {
+    border: 1px solid var(--primary-color);
+    box-shadow: none;
+    border-radius: var(--border-radius);
+    font-weight: 600;
+    display: inline-block;
+    padding: 6px 16px;
+    font-size: var(--font-size-small);
+    outline: 0px;
+    line-height: 1;
+    text-align: center;
+    white-space: nowrap;
+    background-color: transparent;
+    transition: background-color 0.2s ease 0s;
+    user-select: none;
+    cursor: pointer;
+  }
+
+  .m-btn:hover {
+    background-color: var(--primary-color);
+    color: var(--primary-color-invert);
+  }
+
+  .thin-border {
+    border-width: 1px;
+  }
+
+  .dropdown {
+    color: var(--nav-hover-text-color);
+    border-color: var(--nav-accent-color);
+    background-color: var(--nav-hover-bg-color);
+    border-radius: var(--border-radius);
+    border-width: 1px;
+    font-family: var(--font-mono);
+    font-weight: 400;
+    font-size: var(--font-size-small);
+    transition: border 0.2s ease 0s;
+    padding: 4px 5px;
+  }
+</style>
+
+<script type="text/javascript">
+  function changeVersion() {
+    selectElement = document.querySelector('#versionSelector');
+    selectedVersion = selectElement.value;
+
+    const doc = document.getElementById('rd');
+
+    if (typeof selectedVersion !== 'undefined') {
+      doc.loadSpec('https://cockroachlabs.cloud/assets/docs/ccapi_spec_' + selectedVersion + '.json');
+    } else {
+      doc.loadSpec('https://cockroachlabs.cloud/assets/docs/ccapi_spec_latest.json');
+    }
+  }
+</script>
+<div class="apidocs">
+  {% comment %}Replace with remote URL once it is complete{% endcomment %}
+  <rapi-doc id="rd"
+            spec-url = "https://cockroachlabs.cloud/assets/docs/ccapi_spec_latest.json"
+            theme = "light"
+            text-color = "#242A35"
+            primary-color = "#6933FF"
+            nav-bg-color = "#fff"
+            show-header = "false"
+            regular-font = "'SourceSansPro-Regular', sans-serif"
+            mono-font = "SFMono-Regular, Menlo, Monaco, Consolas, 'Liberation Mono', 'Courier New', monospace"
+            show-method-in-nav-bar = "as-colored-text"
+            allow-server-selection = "false"
+            use-path-in-nav-bar = "false"
+            schema-description-expanded = "true"
+            allow-spec-file-download = "true">
+    <div slot="nav-logo" style="display: flex; align-items: center; justify-content: center;"> 
+      <img src = "https://d33wubrfki0l68.cloudfront.net/1c17b3053b29646cdddc53965186a02179b59842/69991/docs/images/cockroachlabs-logo-170.png" style="width:170px; margin-right: 20px" alt="Cockroach Labs logo"> <span style="color:#fff"> <b>nav-logo</b> slot </span>
+    </div>
+    <div slot="overview">
```

Comment:
I love the version switcher, but I feel like it should be higher precedence on the page. I'm imagining having it in the sidebar so that it's always visible, because it's so important that people don't get the wrong version.